config/samples/dataplane_v1beta1_openstackdataplanenodeset_ceph_hci.yaml

```diff
@@ -25,8 +23,6 @@ spec:
         ctlplane_mtu: 1500
         ctlplane_subnet_cidr: 24
         dns_search_domains: []
-        timesync_ntp_servers:
```

Comment:
Why're we removing this? Isn't it a valid default?

Reply:
Just removed it because it is the default, so doesn't need to be explicitly set here:
https://review.rdoproject.org/zuul/build/4583a2fda8314470917efb3e048bd2d8/log/ci-framework-data/logs/192.168.122.100/etc/chrony.conf#7